tests/test_georaster.py

```diff
@@ -1798,22 +1836,42 @@ def test_resampling_str(self) -> None:
         img3b = img1.reproject(img2, resampling=rio.warp.Resampling.q1)
         assert img3a == img3b
 
-    def test_polygonize(self) -> None:
+    @pytest.mark.parametrize("example", [landsat_b4_path, aster_dem_path])  # type: ignore
+    def test_polygonize(self, example: str) -> None:
         """Test that polygonize doesn't raise errors."""
-        img = gr.Raster(self.landsat_b4_path)
 
-        value = np.unique(img)[0]
+        img = gr.Raster(example)
 
-        pixel_area = np.sum(img == value) * img.res[0] * img.res[1]
+        # -- Test 1: basic functioning of polygonize --
 
-        polygonized = img.polygonize(value)
+        # Get unique value for image and the corresponding area
+        value = np.unique(img)[0]
+        pixel_area = np.sum(img == value) * img.res[0] * img.res[1]
```

Comment:
Could you change this to `np.count_nonzero()`? I know they're synonymous in the case of booleans, but "count nonzero" makes things easier to understand in the long term in my opinion.

Reply:
Done